### ISSUE TITLE
Documentation review

### ISSUE THREAD

**Issue by Firstyear** [kanidm/kanidm#661]
Can you please review the tech-writing / docs :)

**Comment by yaleman** [comment]
Does this still need actioning? 😄

**Comment by Firstyear** [comment]
Yes it does, I really wanted @Charcol or @cjschroder to have a look if possible since both of them are exceptional tech writers and could really help. @micolous also could be good to read over this too. :)
